js/letters.js

```diff
@@ -16,7 +16,18 @@ var counter = {
 }
 
 function countLetters(counter, sample_text){
-  // FIX ME
+  const currentChar = sample_text.charAt(0).toLowerCase();
```

Comment:
This should come after the base case, then it won't be executed unless the string length is greater than zero.